app/actions/AliasesActions.js

```diff
@@ -1,6 +1,6 @@
 import * as types from '../constants/AliasesConstants'
 import db from '../config/database.js'
-import { generateKey } from '../../utils/pgp'
+import { generateKey } from '../utils/pgp'
```

Comment:
actually, it seems this whole file is not actually used and has broken imports. I assume it's because it is/was still in development